packages/slate-base64-serializer/package.json

```diff
@@ -4,33 +4,30 @@
   "version": "0.2.21",
   "license": "MIT",
   "repository": "git://github.com/ianstormtaylor/slate.git",
-  "main": "./lib/index.js",
+  "main": "dist/cjs/slate-base64-serializer.js",
+  "module": "dist/es/slate-base64-serializer.js",
+  "browser": "dist/umd/slate-base64-serializer.js",
+  "browserMin": "dist/umd/slate-base64-serializer.min.js",
   "files": [
-    "dist/",
-    "lib/"
+    "dist/cjs/slate-base64-serializer.js",
+    "dist/es/slate-base64-serializer.js",
+    "dist/umd/slate-base64-serializer.js",
+    "dist/umd/slate-base64-serializer.min.js"
```

Comment:
Same goes for all of these files in the other packages too, to match the `dist/` and `lib/` structure.